pkg/rego/load.go

```diff
@@ -84,6 +84,11 @@ func (s *Scanner) LoadPolicies(loadEmbedded bool, srcFS fs.FS, paths []string, r
 		s.policies = make(map[string]*ast.Module)
 	}
 
+	if s.policyFS != nil {
+		s.debug.Log("Overriding filesystem for policies!")
```

Comment:
spurious exclamation mark

Reply:
Yeah I got a bit overexcited about the override.